components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -2,55 +2,63 @@ import { useState } from "react";
 import { Center, Flex, GridItem, SimpleGrid, Text } from "@chakra-ui/react";
 import DaysOfWeekButton from "./DayOfTheWeekButton";
 import LinkButton from "../LinkButton/LinkButton";
+import useForm from "../../components/FormProvider";
 
 export default function DaysOfTheWeekContainer({
   setNumberOfDays,
   saveDataAndLogs,
   customHref,
   disabledDays,
 }) {
+  const { answers, _ } = useForm();
+  const [wfhDays, __] = useState(answers.wfhDays);
+  const [onsiteDays, ___] = useState(answers.onsiteDays);
+
   // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
   const [daysOfTheWeek, setDaysOfTheWeek] = useState([
     {
       id: 0,
       day: "Monday",
-      isSelected: false,
+      isSelected: wfhDays.includes("Monday") || onsiteDays.includes("Monday"),
```

Comment:
Hi @hqtan , look great, just wonder if we shoould refer to **day** key within the object instead of hard coding it?

Reply:
Hi @OlaShabalina :wave: 
How does one refer to `day` key within object? Were you suggesting something like this?
```suggestion
      isSelected: wfhDays.includes(day) || onsiteDays.includes(day),
```

Reply:
I wonder if it will work this way. I have issues pulling the code to my machine.. 

We could place this condition within a function above the state declaration:
```
const someFunctionName = (day) => wfhDays.includes(day) || onsiteDays.includes(day);
```
then use it within the state:
```
 isSelected: someFunctionName(this.day);
```
What do you think? Happy with the current solution too, just thinking 😄

Reply:
Hi @OlaShabalina ,

I could possibly replace lines 18-64 with something like this:
```
  const isSelectedDay = (day) => wfhDays.includes(day) || onsiteDays.includes(day);
  const sevenDays = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];

  const [daysOfTheWeek, setDaysOfTheWeek] = useState(sevenDays.map((day, idx) => {
    return {
      id: idx,
      day: day,
      isSelected: isSelectedDay(day),
      isDisable: false,
    }
  }));
```
And that ___might___ work... 

The code above does look more terse... but it might not be as easy to understand as the current code (although the current code is more repetitive, with harded coded values...)

Since what we have now currently works; If it's ok with you, let's keep it that way for now; we can always refactor after the release if we feel the need to, k?

Reply:
no worries at all, happy to proceed as it is! 👍